development/kops/install_requirements.sh

```diff
@@ -49,9 +53,12 @@ if ! command -v kubectl &> /dev/null
 then
     echo "kubectl could not be found"
     
-    KUBECTL_VERSION=v1.18.9
-    KUBECTL_PATH=/usr/local/bin/kubectl
-    curl -sSL "https://distro.eks.amazonaws.com/kubernetes-1-18/releases/1/artifacts/kubernetes/${KUBECTL_VERSION}/bin/linux/amd64/kubectl" -o ${KUBECTL_PATH}
+    KUBECTL_VERSION=$(cat ../../projects/kubernetes/kubernetes/${RELEASE_BRANCH}/GIT_TAG)
```

Comment:
Can we use a variable pointing to root of the repo here (`$REPO_ROOT/projects/...`), or better yet make the path relative to BASEDIR?

Reply:
Sounds good. Done.